pkg/controller/directvolumemigration/pvcs.go

```diff
@@ -72,7 +75,35 @@ func (t *Task) createDestinationPVCs() error {
 	return nil
 }
 
-func (t *Task) getDestinationPVCs() error {
+func (t *Task) getDestinationPVCs() (bool, error) {
```

Comment:
func name should change to `areDestinationPVCsBound()` or something that indicates why we are returning a bool